deploy/prod/main.tf

```diff
@@ -222,7 +251,47 @@ resource "aws_ecs_service" "git" {
 
 }
 
+resource "aws_ecs_service" "github" {
+  name            = "insights-github"
+  cluster         = aws_ecs_cluster.insights-git-cluster.id
+  task_definition = aws_ecs_task_definition.insights-connector-github-task.arn
+  desired_count   = 1
+  launch_type                        = "FARGATE"
+  scheduling_strategy                = "REPLICA"
+  network_configuration {
+    security_groups = [aws_security_group.security_group.id]
+    subnets = [aws_subnet.main.id]
+    assign_public_ip = true
+  }
+}
 
+resource "aws_ecs_service" "jira" {
+  name            = "insights-jira"
+  cluster         = aws_ecs_cluster.insights-git-cluster.id
```

Comment:
`/s/aws_ecs_cluster.insights-git-cluster.id/aws_ecs_cluster.insights-jira-cluster.id/`?

Reply:
renamed the cluster to a common for all name.